src/allotropy/allotrope/schema_parser/schema_cleaner.py

```diff
@@ -271,7 +269,7 @@ def _dereference_value(self, value: dict[str, Any]) -> dict[str, Any]:
             ):
                 return result
             # NOTE: assumes ref is cleaned.
-            # NOTE: we do not futher deference values because we don't want to remove all
+            # NOTE: we do not further dereferencevalues because we don't want to remove all
```

Comment:
tiny nit: `dereferencevalues` missing a space (don't bother fixing if it's the only thing)

Reply:
Sounds good 🙇 , I'll make a note to fix.